packages/api-server/src/watch.ts

```diff
@@ -20,6 +20,7 @@ import {
   ensurePosixPath,
   getConfig,
   resolveFile,
+  isUsingExperimentalServerFile,
```

Comment:
I hate to do this this, but once this goes non-experiment, this name is going to be confusing.

`useServerFile` or `hasServerFile` or "isServer" or something else is going to be more lasting. we'd have a PR to change the name later.